internal/pkg/describe/service.go

```diff
@@ -35,12 +35,14 @@ type ConfigStoreSvc interface {
 	ListEnvironments(appName string) ([]*config.Environment, error)
 	ListServices(appName string) ([]*config.Workload, error)
 	GetWorkload(appName string, name string) (*config.Workload, error)
+	ListJobs(appName string) ([]*config.Workload, error)
 }
 
 // DeployedEnvServicesLister wraps methods of deploy store.
 type DeployedEnvServicesLister interface {
```

Comment:
What do you think of renaming this to `DeployedEnvWorkloadsLister`?

Reply:
I thought of doing so but there are a lot of places where we need to change if we change it to `DeployedEnvWorkloadsLister`. for example, impacted places are - 
1. env.go
2. env_test.go
3. lb_web_service.go
4. lb_web_service_test.go
5. mock_service.go
6. rd_web_service.go
7. rd_web_service_test.go
8. service.go
9. worker_service.go

If we really be touching so many files as a part of `env show` implementation?